core/src/main/java/org/apache/accumulo/core/clientImpl/ThriftTransportPool.java

```diff
@@ -94,17 +104,20 @@ private ThriftTransportPool(LongSupplier maxAgeMillis) {
       } catch (TransportPoolShutdownException e) {
         log.debug("Error closing expired connections", e);
       }
-    });
+    };
```

Comment:
With this change the server will terminate if this thread exits for any reason. The AccumuloUncaughtExceptionHandler will be invoked for any uncaught exception that causes this thread to exit. Because InterruptedException is caught, the UEH will not be invoked, and nothing will be logged. We likely want to add logging in the InterruptedException case. The logging in AccumuloUncaughtExceptionHandler should be sufficient for other RuntimeException's.

Reply:
Would be nice to add a warn log when the interrupt exception happens, the log message could state it was ignored.  Warn makes it more likely to see the event was ignored, in case we want to change how that exception is handled in the future.